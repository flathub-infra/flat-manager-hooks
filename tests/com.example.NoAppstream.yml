id: com.example.NoAppstream
runtime: org.freedesktop.Platform
sdk: org.freedesktop.Sdk
runtime-version: '22.08'
command: main
modules:
- name: test1
  buildsystem: simple
  build-commands:
  - gcc -o main main.c
  - install -D main /app/bin/main
  sources:
  - type: file
    path: main.c
