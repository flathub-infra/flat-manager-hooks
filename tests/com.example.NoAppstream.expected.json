{
  "diagnostics": [
    {
      "refstring": "app/com.example.NoAppstream/x86_64/master",
      "is_warning": false,
      "info": {
        "category": "FailedToLoadAppstream",
        "data": {
          "path": "files/share/app-info/xmls/com.example.NoAppstream.xml.gz",
          "error": "File does not exist"
        }
      }
    }
  ]
}
